einterfaces/metrics.go

```diff
@@ -39,6 +39,8 @@ type MetricsInterface interface {
 	IncrementWebSocketBroadcast(eventType string)
 	IncrementWebSocketBroadcastBufferSize(hub string, amount float64)
 	DecrementWebSocketBroadcastBufferSize(hub string, amount float64)
+	IncrementWebSocketBroadcastUsersRegistered(hub string, amount float64)
+	DecrementWebSocketBroadcastUsersRegistered(hub string, amount float64)
```

Comment:
Out of curiosity, are we ever planning to increment/decrement more than 1 at a time? Does such a use case exist?

Reply:
Right now is not contemplated but given that is a metric and it could be increased or decreased in cases where we're removing a hub or unregistering a bunch of users at once don't do any harm having it flexible IMHO

Reply:
I had actually thought about it in the previous PR, and decided it's okay to leave the amount configurable.